codex-rs/tui/src/chatwidget.rs

```diff
@@ -435,6 +456,13 @@ impl ChatWidget<'_> {
                     changes,
                 ));
             }
+            EventMsg::PatchApplyEnd(event) => {
+                self.add_to_history(HistoryCell::new_patch_apply_end(
+                    event.stdout,
```

Comment:
I would just pass the whole `PatchApplyEndEvent` rather than pull the fields off here.